src/components/ProjectLabel.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import ProjectIcon from './ProjectIcon'
+import styles from './ProjectLabel.module.scss'
+
+const ProjectLabel = ({ name, ticker, className }) => (
+  <div className={cx(styles.label, className)}>
+    <ProjectIcon
+      name={name}
+      ticker={ticker}
+      size={20}
+      className='project-icon-default'
```

Comment:
Why not css module style? @devanshu-santiment

Reply:
I realised we don't need that anyway

Reply:
If there is no need for that anyway, why it's not removed?